### ISSUE TITLE
Change how the Irene Bower missions tie into the FW campaign

### ISSUE THREAD

**Issue by Amazinite** [endless-sky/endless-sky#9801]
### Problem Description

The Care Package to South mission with Irene Brower, have follow up missions that attached themselves directly to missions in the main campaign. That is, upon finishing certain missions (and therefore certain conversations) related to the main campaign, you are (at least intended to) immediately greeted by a follow-up conversation with Irene Brower. This occurs in "Care Package to South 2b" and "Care Package to South 2a". But as noted by #9780, the offering of these missions is not working as intended at the moment.

### Related Issue Links

#9780

### Desired Solution

Ditch the idea of having these missions attached directly to main campaign missions. I think that if we want to have missions that you do prior to the main campaign that can add extra flavor during the campaign, those follow-up missions should offer randomly during certain stretches of the campaign without offering alongside a main mission.

### Alternative Approaches

Turn the missions into branches of the missions they're intended to offer alongside.

**Comment by bene-dictator** [comment]
@danaris may want to look at this since this mission was originally written by them.

Personally I think that the branch idea would be the simplest option, though it does feel like giving these missions preferential treatment to add content from them to the campaign files itself.

Additionally, it's possible that, if other missions want to base themselves off the text in the FW campaign, they'd be blocked from using that specific mission (and we can't just use branches for everything). The better option is to do as the issue suggests.

This is also something I'll have to change for Adelita since the next missions in that chain append to the end of the Electron Beam mission and the Battle of Rastaban.

**Comment by danaris** [comment]
I agree that overall, elevating these to be branches of main-story missions is giving them a preferential treatment they do not deserve.

I would prefer _not_ to have the story linkage to the main storyline events lost, because I've already gotten several compliments on showing the toll and the aftermath of those in more tangible ways. However, that doesn't necessarily mean they have to be linked quite so closely—and if the mechanics are not linking them correctly, then that definitely needs to be fixed. (They worked correctly when I tested them, but it's not surprising to me that that's proven fragile.)

I wonder if it would be better, in the long run, to add a more robust way of having one mission "follow on from" the end of another, guaranteeing that, if you meet the conditions, it will trigger immediately after its "on complete"? This could be potentially useful for main-story missions, as well.

For the short term, Care Package 2B could come up any time after FW Pirates Diplomacy 4 when you land on a FW planet; it would just need the text adjusting to have the opening be more generic, and the time frame be more ambiguous. I'd be happy to make that adjustment.

For Care Package 2A....I'm not sure. That one really only makes sense pretty immediately after the battle, so I'm not sure how it could be reworked to appear randomly during that stretch of the campaign. (I'm also not entirely sure how it could be guaranteed to come up _at some point_ during that stretch of the campaign, without being guaranteed to come up at a _particular_ point within it...) If I were going to try to rework this one in some way, I'd need to go back into the mission files and re-familiarize myself with both the narrative and the mission structure in this part of the campaign.

**Comment by bene-dictator** [comment]
Relevant PR: #9924
